packages/contracts-bedrock/test/L2/CrossL2Inbox.t.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
+import { Reverter, ConfigurableCaller } from "test/mocks/Callers.sol";
+import { ICrossL2Inbox } from "src/L2/ICrossL2Inbox.sol";
+
+// Libraries
+import { SafeCall } from "src/libraries/SafeCall.sol";
+
+import { L1Block } from "src/L2/L1Block.sol";
+
+contract CrossL2InboxTest is CommonTest {
+    address depositor;
+
+    function setUp() public virtual override {
+        super.setUp();
+        depositor = l1Block.DEPOSITOR_ACCOUNT();
+    }
+
+    function testFuzz_executeMessage_succeeds(
+        bytes calldata _msg,
+        ICrossL2Inbox.Identifier calldata _id,
+        address _target
+    )
+        external
+        payable
+    {
+        // need to make sure address leads to successfull SafeCall by executeMessage
+        // TODO: how to make sure gasLeft coincides wtih executeMessage?
+        bool success = SafeCall.call({ _target: _target, _gas: gasleft(), _value: msg.value, _calldata: _msg });
```

Comment:
i think an easier way to do this is to assume no code in target: `vm.assume(_target.code.lenth == 0);`

Reply:
This may fail for precompiles so we may also need to use the forge-std `assumeNoPrecompiles`

Reply:
alternatively, we can just use `vm.etch(_target, address(0).code)`